### PR TITLE
Mozsprint 2018

### PR DESCRIPTION
**Pull request by mssososophie**
The purpose of this pull request is to add our document which features detailed instructions, guides and examples on how a someone experienced in ML could use satellite technology and data processing tools to automatically map the unmapped areas of Tanzania.

This is an incomplete document co-authored by Sophie Ashcroft (Independent), Rohini Joshi, Elliott Polzin, Robert Lyon, Therese Cantwell, Joe Fennell(all of JBCA, University of Manchester) as part of Mozsprint 2018 hosted at Manchester University by Rachael Ainsworth

### CODE REVIEW

**Comment by janetchapman** [comment]
Brilliant! Thank you so much!

File: mozsprint2018DataProcessingSentinel.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+*Document target user*
+
+This is written for individuals confindent and compitent in Machine Learning technologies and systems.
+
+### Document Goal
+
+After reading into the context and scope of the Crowd2Map project we found that with the time and human constraits we are under, the best way for us to contribute would be to create a set of recommendations for how in the future contributors can automate mapping by using a combination of satallite and data processing technologies.
+
+Below is a proposed workflow of what data should be collected, from where, how to train a K-means algorithm on it and what this proposed solution will deliver. Finally there is further information on how this process has worked in practise on a map of Peurto Rico.
+
```
Comment (janetchapman) [inline mozsprint2018DataProcessingSentinel.md:25]: What does K-means mean please? Does anyone have any links to mapping examples?

Comment (mssososophie) [reply mozsprint2018DataProcessingSentinel.md:25]: https://en.wikipedia.org/wiki/K-means_clustering

**Comment by mssososophie** [comment]
It's not complete but it'd be a leap in the right direction for any future developer who would have the time and skill to dedicate towards automating the mapping needed. 
There are so many papers and projects already out there that will be able to help you with this.
Joe Fennell says that it's quite possible that you'd be able to find a grand funding to get this project going, please e-mail him for more details.

**Comment by janetchapman** [comment]
Thank you so much, I will do.

Best wishes
Janet
________________________________

**Pull request merged**